cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function is used with a `namespace` argument, but the `namespace` should be defined in the included `urls.py` files of the respective apps (`cinema` and `user`). Ensure that the `urls.py` files in these apps have a `namespace` defined in their `app_name` variable.